build.gradle.kts

```diff
@@ -58,7 +58,10 @@ kotlinDslPluginOptions {
 }
 
 dependencies {
-    implementation("software.amazon.awssdk:s3:2.17.106") {
+    implementation(platform("software.amazon.awssdk:bom:2.17.267"))
+    // Needed to automatically enable AWS SSO login
+    implementation("software.amazon.awssdk:sso")
```

Comment:
```suggestion
    implementation("software.amazon.awssdk:sso") {
        because("Needed to automatically enable AWS SSO login, see https://stackoverflow.com/a/67824174")
    }
```